packages/core/src/HealthwiseLogo/index.js

```diff
@@ -8,200 +8,33 @@ class HealthwiseLogo extends Component {
 
     return (
       <svg
-        viewBox="0 0 165 74"
         className={classNames('hw-healthwise-logo', className)}
         {...otherProps}
```

Comment:
Should this be injected as the last prop?  I think the others below it would override it in the case that someone passed in `width` or `height`, for example.

Reply:
yep - good point! changing